tests/foreman/ui/test_user.py

```diff
@@ -307,6 +308,88 @@ def test_positive_create_product_with_limited_user_permission(
         assert newsession.product.search(product_name)[0]['Name'] == product_name
 
 
+@pytest.mark.rhel_ver_match('8')
```

Comment:
Missing tier decorator

Reply:
Do we really need tier decorator now? What is the purpose to use it. I thought it has become obsolete now.

Reply:
Frankly, I don't know. At least for consistency, I would expect it to be there. Best to ask CI team what is expected regarding tier decorators.

Reply:
I remember a discussion over it and I think it is not required but would like to have some thoughts on it. 
@devendra104 wdyt?